### DISCUSSION TITLE
we need to give more autonomy to new contributors (members)

### DISCUSSION THREAD

**Discussion by avelino** [prest/prest#758] [General]
I feel that I am blocking contributions from new contributors, and this has made me feel powerless (my professional life is more hectic than I would like).

We need to make our contribution environment more fluid and give more autonomy to the people who want to contribute.

How do I imagine solving this problem?

- improve our testing environment
- improve the pull request and issue templates
- don't depend only on me and @arxdsilva for revision — @vmesel and @cuducos are contributing a lot (@fabriziomello also contributes a lot of ideas from the postgres world), if I haven't already spoken welcome, I count on you in prest core team
- there is a time limit for reviewing contributions
- review the open issues and understand what makes sense to keep the open
- tagging issues 
- revisit [contribution policy](https://docs.prestd.com/contribution/)

> I don't know if this is enough to improve, i would like to hear more people's opinion about this subject

**[also contribute](https://github.com/prest/prest/discussions/759)**

**Why am I writing this thread?**

Seeing this relatively simple (PR) contribution take more than **3 weeks to merge hurts my heart** #750

**Comment by cuducos** [comment]
+1 especially for _improve our testing environment_.

A playground would also be nice: a pREST instance running with its database, configs, and source open to the public. That's nice when people try to understand parts of the codebase they haven't touched yet.

**Comment by vmesel** [comment]
Thank you for bringing this up @avelino!

I think this PR is being blocked a lot because we don't have a good set of testing environments and patterns, with this in mind I really think we could write down contribution requirements inside the PR template, so people could observe those sets of rules and infuse them in their contribution.

A good example of [Development Guidelines is Django's](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/)

Also, this PR delay is related to another thing: I'm a newbie to Golang and its patterns, so I'm still on the learning curve. Probably, if I were more senior to Go, we would be able to cut half of the time.

> A playground would also be nice: a pREST instance running with its database, configs, and source open to the public. That's nice when people try to understand parts of the codebase they haven't touched yet.

On the playground side, it would really be cool to have it, and it would ease the showcase for new companies and teams, onboarding them better and quicker.
